convert/convert.go

```diff
@@ -12,9 +12,9 @@ import (
 	"github.com/docker/docker/runconfig/opts"
 	"github.com/docker/go-connections/nat"
 	"github.com/docker/go-units"
-	"github.com/docker/libcompose/utils"
 	"github.com/rancher/rancher-compose-executor/config"
 	"github.com/rancher/rancher-compose-executor/project"
+	"github.com/rancher/rancher-compose-executor/utils"
```

Comment:
If you import `github.com/docker/libcompose/utils` instead you won't need to add functions to our `utils` package. Though I'd like to remove the libcompose import entirely at some point.